ui/src/js/viewModels/orgAdmin.js

```diff
@@ -7,436 +7,444 @@
 /*
```

Comment:
The diffs in this file aren't showing up - it is telling me everything has changed. What has changed in reality?

Reply:
We commented out
```
return $.when(restClient.doPost('/rest/organizations/' + self.userOrgId() + '/user_organizations', userOrgData)
    .then(
        success = function (response) {
            self.getUserOrgData(self.userOrgId());
            console.log("user data posted");
        })
);
```
Lines ~308-315

The reason it shows so many differences is because of the autoformatter in VSCode.